rule/engine_glob.py.go

```diff
@@ -0,0 +1,95 @@
+package rule
```

Comment:
A particular reason for naming this file `.py.go`? If not, let's rename it to `engine_glob.go` :)